test/runtests.jl

```diff
@@ -68,6 +68,120 @@ end
                    8.848886704358202, 2.8697881579099085, 11.15107375630744, 11.8392578374927])
 end
 
+@testset "rankdeficient-glm" begin
+    Random.seed!(9999)
+    df = DataFrames.DataFrame()
+    num_rows = 100
+    df[:feature_1] = randn(num_rows)
+    df[:feature_2] = 2 .* df[:feature_1]
+    df[:feature_3] = randn(num_rows)
+    df[:label_1] = round.(Int, rand(num_rows))
```

Comment:
This should probably just be `rand(0:1, num_rows)`.

Reply:
Done.